### DISCUSSION TITLE
Is it necessary for flutter_rust_bridge to have a version dependency on the flutter sdk greater than `3.19.6`?

### DISCUSSION THREAD

**Discussion by Jhonfunk** [fzyzcjy/flutter_rust_bridge#2141] [General]
Windows 7 is no longer supported in Flutter versions greater than 3.19.6.

**Comment by fzyzcjy** [comment]
I guess 3.19 may work. Firstly try to run it and see what error we get. I guess you need to `web: false` to avoid using new Dart language features.
